### DISCUSSION TITLE
Version 7.0 development

### DISCUSSION THREAD

**Discussion by HuubMons** [HuubMons/HuMo-genealogy#30] [General]
Hi Barcha,

I'm not used to working together in GitHub. Still learning...
Is it ok if I also do some merges into dev7? I don't want to interrupt your updates, I really appreciate them!

Oh, and if you like it do enter your name in the credits!

**Comment by ghost** [comment]
Any developer can make a 'pull request', only you are able to do the merge, if there is an impossibility in the code to execute a change, git will tell you. And yes, you can port your work to any branch.

**Comment by HuubMons** [comment]
I didn't notice the latest pull request was to MASTER.
Now I have to try to restore this.

Revert was succesfull. But pull request is closed now...

**Comment by ghost** [comment]
huummm why you had this master merge .... my first was  me-7.0-dev tu home-7.0-dev... I have to take care of that.
Sorry about this mistake

**Comment by HuubMons** [comment]
I'm reading manuals how to reopen the pull request. The reopen button is missing.

**Reply by ghost** [reply]
Let this one closed.

**Comment by HuubMons** [comment]
I probably have to restore the master to the exact status before this merge. Then the re-open option could be visible.

The latest changes were done at the GitHub website, before that I was using my Raspberry Pi, using github CLI.
I will try getting the hash numbers of the previous version in CLI.

**Reply by ghost** [reply]
Let like that. I will take care next time.

**Comment by HuubMons** [comment]
https://cli.github.com/manual/gh_pr_reopen
Stupid: I can't find the GitHub CLI.

Oow. It must be installed...

**Comment by HuubMons** [comment]
Ok, I give up. It's almost impossible to reopen a merged pull request.

**Comment by HuubMons** [comment]
Remark:
I changed **A LOT** in the editor files in the last weeks.

I have to merge all my changes in these files:
admin/include/editor.php
admin/include/editor_cls.php
admin/include/editor_event_cls.php
admin/include/editor_inc.php
This file will be removed in the next version: HuMo-genealogy/admin/include/editor_sources.php

Other files can be changed without problems.

I do have a problem: almost every line in these files is changed! 
Is it possible to run these files through your validator (can be fond in dev 6.0)?
admin/include/editor.php
admin/include/editor_cls.php
admin/include/editor_event_cls.php
admin/include/editor_inc.php

Then it easier to use the merge programs.
